pom.xml

```diff
@@ -119,11 +120,37 @@
             <artifactId>spring-cloud-stream-binder-rabbit</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>io.r2dbc</groupId>
+            <artifactId>r2dbc-postgresql</artifactId>
+            <version>${r2dbc-postgresql.version}</version>
```

Comment:
Yes, they are already included in springboot dependencyManagement so I removed everything. This evens leads to using 0.8.8.RELEASE instead of 0.8.4.RELEASE for h2, which is good.

Reply:
I copied this from our other r2dbc implementations but shouldn't have